Automattic/wp-calypso
# Registrationless checkout: Show the help icon in the checkout page

#### Changes proposed in this Pull Request

**This is one among a set of stacked PRs that implements userless checkout A/B test. For more reading on the userless
checkout project, read pau2Xa-1hW-p2.**

* The inline help icon will be loaded but kept hidden beneath the Google Recaptcha logo. This way, clicking the "Read more about plans and purchases" link in the Purchase details section will open the help popover without displaying the floating help icon to the user.

#### Testing instructions



* Go to /start/onboarding-new and select a paid plan to open the checkout page.
* Verify that the Google Recaptcha logo is on the bottom right and the floating help icon is not visible.
* Click `Questions? Read more about plans and purchases` from the order summary and verify that the inline help popover opens.

![uc](https://user-images.githubusercontent.com/1269602/88935213-21fbbb00-d29f-11ea-94a3-00952365fedd.gif)


* Confirm mobile view looks okay

<img width="372" alt="Screenshot 2020-07-30 at 7 57 07 PM" src="https://user-images.githubusercontent.com/1269602/88934974-dfd27980-d29e-11ea-8a3f-453658dd4b32.png">
